static/js/zamboni/static_theme.js

```diff
@@ -11,7 +11,10 @@ $(document).ready(function() {
         function getFile() {
             file_selector = $wizard.find('#header-img')[0];
             file = file_selector.files[0];
+						if (!file.name.match(/.(jpg | jpeg | png | gif | apng)$/i))
```

Comment:
This works great (apart from missing `.svg` off) but it'd be good to re-use the `accept` attribute from the html so the list of acceptable files is only defined in one place.  
How about:
`if (file && $wizard.find('#header-img').attr('accept').split(',').indexOf(file.type)== -1)` ?
It uses file.type to get content type which should hopefully be more robust too.

Reply:
Wow! That looks like a more clean and efficient way of doing the task. 
Thanks :)